### ISSUE TITLE
Can't update from keyserver.

### ISSUE THREAD

**Issue by cpdyj** [open-keychain/open-keychain#2280]
Can't update from keyserver.
## Expected Behavior

Update.
## Current Behavior

Update fail. A certification exception.
```
Network error (CertificateException) for 'hkps://keyserver.ubuntu.com'. Check your Internet connection! 
```
## Steps to Reproduce (for bugs)

1. ![IMG_20180304_212858.jpg](https://i.loli.net/2018/03/04/5a9bf65116e2f.jpg)

## Your Environment

* Android Version: Android 5.0.2
* Device Model: Vivo y35a/armeabi-v7a
* OpenKeychain Version: 4.9(49001)
* From Google Play or F-Droid?: F-Droid

Thank you.

**Comment by dschuermann** [comment]
This issue is fixed in 4.9.1 which should be available soon on F-Droid.

**Comment by dschuermann** [comment]
This issue is fixed in 4.9.1 which should be available soon on F-Droid.

**Comment by dschuermann** [comment]
This issue is fixed in 4.9.1 which should be available soon on F-Droid.

**Comment by dschuermann** [comment]
This issue is fixed in 4.9.1 which should be available soon on F-Droid.
